package.json

```diff
@@ -52,7 +51,7 @@
     "test-storybook": "test-storybook",
     "test-storybook:watch": "test-storybook --watch",
     "test-storybook:ci": "start-server-and-test storybook:ci http://localhost:6006 test-storybook",
-    "build-storybook": "storybook build -o 'docs'",
+    "build-storybook": "storybook build --docs -o 'docs'",
```

Comment:
Adding --docs means only build docs files  ie: .mdx files

Reply:
This doesn't effect Chromatic, right?

Reply:
That's a great question. It will. looking at the chromatic github job I see this command for building stories uses our build-storybook command. Let me see how to work around that

```
Building your Storybook
    → Running command: npm run build-storybook -- --output-dir /tmp/chromatic--1973-bDmvKOHR2i1r

```